docs/source/hyper-parameter-search.rst

```diff
@@ -22,12 +22,12 @@ Dask-ML's :ref:`*incremental* hyperparameter optimizers <hyperparameter.incremen
 Drop-In Replacements for Scikit-Learn
```

Comment:
This needs some more work. Spend time integrating with #432.

Reply:
I decided to remove all the documentation changes. I'll implement them in #432, and focus more there.